src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -566,7 +567,7 @@ AmendmentTableImpl<AppApiFacade>::doVoting (Ledger::ref lastClosedLedger,
         Serializer s;
         trans.add (s, true);
 #if RIPPLE_PROPOSE_AMENDMENTS
-        SHAMapItem::pointer tItem = std::make_shared<SHAMapItem> (txID, s.peekData ());
+        std::shared_ptr<SHAMapItem> tItem = std::make_shared<SHAMapItem> (txID, s.peekData ());
```

Comment:
auto here. I'm not necessarily in the 'auto almost everywhere' camp, but when the type is obvious - like with make_shared - I prefer auto.